### ISSUE TITLE
 [Bug]: Hyprland crashes on startup in Fedora 41

### ISSUE THREAD

**Issue by dani0221** [JaKooLit/Fedora-Hyprland#153]
## Bug Report

## System information

- Operating system: Fedora 41
- Hyprland version: [ aquamarine 0.4.4]
- Desktop environment: [Gnome v47]

## steps to reproduce
1. Start Hyprland.
2. After starting, the program does not work
3. When trying to start the program manually, it crashes immediately and displays an error message.

## Expected behavior
Hyprland should start correctly and display the desktop environment without errors
## Actual behavior
Hyprland crashes during startup and displays an error message related to 'wl_registry. The program terminates with an IOT instruction error.
Hyprland has crashed :(
## error output (wl_registry#2: error 0: invalid version for global wl_seat (15): have 8, wanted 9
terminate called after throwing an instance of 'std::runtime_error'
  what():  CBackend::create() failed!
Hyprland has crashed :( Consult the crash report at /home/angel-dev/.cache/hyprland/hyprlandCrashReport11565.txt for more information.
[1]    11565 IOT instruction (core dumped)  Hyprland)
## files of crash
[hyprlandCrashReport11565.txt](https://github.com/user-attachments/files/17739298/hyprlandCrashReport11565.txt)

**Comment by JaKooLit** [comment]
just to be clear, you tried to start via log in manager? sddm.. gdm.. etc


and where is the output from inxi?

**Comment by JaKooLit** [comment]
running on a hardware of a VM?

**Comment by dani0221** [comment]
No, the program is not running in a virtual machine. I'm running Fedora Linux directly on my physical laptop

**Comment by dani0221** [comment]
I already solved it

**Comment by akhil-hari** [comment]
@dani0221 
Can you explain how you solved this issue?

**Comment by dani0221** [comment]
@akhil-hari  yes, It was starting in gnome and giving problems, but I ran it from a tty (unix)

**Comment by JaKooLit** [comment]
just choose the none uswm
